cmd/frontend/internal/app/ui/router.go

```diff
@@ -212,6 +214,11 @@ func initRouter(db dbutil.DB, router *mux.Router) {
 	router.Get(routeHome).Handler(handler(serveHome))
 	router.Get(routeThreads).Handler(handler(serveBrandedPageString("Threads", nil)))
 	router.Get(routeInsights).Handler(handler(serveBrandedPageString("Insights", nil)))
+	router.Get(routeLegacyCampaigns).Handler(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		r.URL.Path = "/batch-changes" + mux.Vars(r)["Path"]
+		// Temporary redirect so at some point we can reuse the /campaigns path, if needed.
+		http.Redirect(w, r, r.URL.String(), http.StatusTemporaryRedirect)
```

Comment:
```suggestion
		http.Redirect(w, r, auth.SafeRedirectURL(r.URL.String()), http.StatusTemporaryRedirect)
```

This would make you safe I believe.

Reply:
Nice, thanks!